packages/webcomponents/src/components/business-form/business-form-stepped.tsx

```diff
@@ -55,16 +43,31 @@ export class BusinessFormStepped {
     this.fetchData = this.fetchData.bind(this);
   }
 
+  componentStepMapping = {
+    0: (formController) => <justifi-business-generic-info formController={formController} />,
+    1: (formController) => <justifi-legal-address-form formController={formController} />,
+    2: (formController) => <justifi-additional-questions formController={formController} />,
+    3: (formController) => <justifi-business-representative formController={formController} />,
+    4: (formController) => <justifi-business-owners formController={formController} />
+  };
+
   componentWillLoad() {
     if (!this.authToken) {
       console.warn(
```

Comment:
I think these should probably be errors instead of warnings since the component won't function without it